src/Plugin/Hal.php

```diff
@@ -155,15 +156,16 @@ public function getController()
      *
      * @return self
      */
-    public function setEventManager(EventManagerInterface $events)
+    public function setEventManager(EventManagerInterface $eventManager)
```

Comment:
It was a PHPStorm complain. Reverting.

Reply:
> Is the change in variable name really necessary here? Elsewhere in the class, we still use `$events`.

Hi @weierophinney, [Psalm is failing](https://github.com/laminas-api-tools/api-tools-hal/actions/runs/3904516372/jobs/6673158204#step:3:529) if I keep the `$events` variable name:

```
Error: src/Plugin/Hal.php:159:59: ParamNameMismatch: Argument 1 of Laminas\ApiTools\Hal\Plugin\Hal::setEventManager has wrong name $events, expecting $eventManager as defined by Laminas\EventManager\EventManagerAwareInterface::setEventManager (see https://psalm.dev/230)
```

Should I ignore or fix the issue?

Reply:
I will rename it to `$eventManager` to fix the psalm error.

Reply:
Ah, okay, that makes sense - because with PHP 8, and the introduction of named arguments, a deviation in argument name when implementing an interface would be considered a potential contractual change.

Go ahead and change it, then - and note in the commit the reason why.

Reply:
The problem comes from `laminas/laminas-eventmanager`

On the [`EventManagerAwareInterface`](https://github.com/laminas/laminas-eventmanager/blob/3.10.x/src/EventManagerAwareInterface.php#L15), the parameter is called `$eventManager` while on [`EventManagerAwareTrait`](https://github.com/laminas/laminas-eventmanager/blob/3.10.x/src/EventManagerAwareTrait.php#L36) is called `$events`

This triggers a Psalm error no matter which variable name I use, so I have [ignored it](https://github.com/laminas-api-tools/api-tools-hal/pull/39/commits/cdfe74b9eb479a320a7061e83f8a0df584c2a5ef#diff-2d20a98059390e83ae58234d4bda07fe9319b822009ce8c4479e0f3ae1943fbfR54). I have also opened an [Issue](https://github.com/laminas/laminas-eventmanager/issues/41) on the library to fix it.

I think the PR is ready to be merged. Please, @weierophinney, let me know if there is anything else to change.